package.json

```diff
@@ -1,36 +1,43 @@
 {
-	"name": "@netresearch/node-magento-eqp",
-	"version": "2.0.0",
-	"description": "Simple API wrapper around the Magento Marketplace EQP API",
-	"main": "dist/index.js",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/netresearch/node-magento-eqp.git"
-	},
-	"types": "dist/index.d.ts",
-	"author": "TheDevMinerTV <tobigames200@gmail.com>",
-	"license": "MIT",
-	"scripts": {
-		"build": "yarn build:lib && yarn build:docs",
-		"build:lib": "tsc",
-		"build:lib:dev": "tsc -w",
-		"build:docs": "typedoc --out docs --entryPoints src/index.ts",
-		"lint": "eslint src --ext .ts",
-		"prepack": "yarn build:lib"
-	},
-	"devDependencies": {
-		"@types/node": "8",
-		"@typescript-eslint/eslint-plugin": "^4.21.0",
-		"@typescript-eslint/parser": "^4.21.0",
-		"eslint": "^7.23.0",
-		"typedoc": "^0.20.35",
-		"typescript": "^4.2.3"
-	},
-	"dependencies": {
-		"axios": "^0.21.1",
-		"tslib": "^2.1.0"
-	},
-	"engines": {
-		"node": ">=8"
-	}
+  "name": "@netresearch/node-magento-eqp",
```

Comment:
Please format this with `esbenp.prettier-vscode`. It should automatically pick up the formatting guidelines (tabs, with 4 spaces, 160 chars print width)